src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -423,8 +423,8 @@ private static void LoadGenesis(ChainSpecJson chainSpecJson, ChainSpec chainSpec
         bool isEip4844Enabled = chainSpecJson.Params.Eip4844TransitionTimestamp != null && genesisHeader.Timestamp >= chainSpecJson.Params.Eip4844TransitionTimestamp;
         if (isEip4844Enabled)
         {
-            genesisHeader.BlobGasUsed = chainSpecJson.Genesis.BlobGasUsed;
-            genesisHeader.ExcessBlobGas = chainSpecJson.Genesis.ExcessBlobGas;
+            genesisHeader.BlobGasUsed = chainSpecJson.Genesis.BlobGasUsed ?? 0;
+            genesisHeader.ExcessBlobGas = chainSpecJson.Genesis.ExcessBlobGas ?? 0;
```

Comment:
simpler would be to change ChainSpecGenesisJson and make those properties non-nullable - they would default to 0

Reply:
From what I understand, pre-cancun we expect it to be null, so we need it nullable. In this case genesis is with cancun EIPs already enabled so we need it as 0 when calculating hash

Reply:
so if eip4844 is disabled when loading genesis, it is null
https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs#L423-L428

Reply:
and we just need it to be 0 if genesis is post-cancun and value is not specified in genesis

Reply:
make it non-nullable in `chainSpecJson.Genesis.BlobGasUsed` so in `ChainSpecGenesisJson`

Reply:
good point. Changing